### PR TITLE
Cloud Changelog - Feb 16

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by marianunez — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -3,12 +3,58 @@
...
+## Release 2022-02-16
+
+### Improvements
+
+#### User Interface (UI)
+ - Users are now able to use the channel switcher to search channels across teams if the users are part of more than one team.
+ - Updated the recommended minimum supported Firefox version to v91+.
+ - Clarified in-product error string "Oops!" as "Unable to continue" for both translators and target audiences in cases where a user doesn't have sufficient permissions to add users or guests.
+ - Removed incorrect in-product string text from the **Send full message contents** email notification option description displayed via **System Console > Site Configuration > Notifications**.
+ - Removed incorrect in-product string text from the **Send full message contents** email notification option description displayed via **System Console > Site Configuration > Notifications**.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:23]: This bullet point appears to be a duplicate of the one directly above it.

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -3,12 +3,58 @@
...
+
+#### User Interface (UI)
+ - Users are now able to use the channel switcher to search channels across teams if the users are part of more than one team.
+ - Updated the recommended minimum supported Firefox version to v91+.
+ - Clarified in-product error string "Oops!" as "Unable to continue" for both translators and target audiences in cases where a user doesn't have sufficient permissions to add users or guests.
+ - Removed incorrect in-product string text from the **Send full message contents** email notification option description displayed via **System Console > Site Configuration > Notifications**.
+ - Removed incorrect in-product string text from the **Send full message contents** email notification option description displayed via **System Console > Site Configuration > Notifications**. 
+ - Added the ability to send unsanitized user to the source user on ``user_updated`` event.
+ - In the compact view, the sender's username is now always shown on posts.
+ - Added a "Create board" button to the Channel Intro section.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:26]:
```suggestion
 - Added a **Create board** button to the Channel Intro section.
```

Comment (cwarnermm) [reply source/install/cloud-changelog.md:26]: I'm not clear on what's we mean by Channel Intro section.

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -3,12 +3,58 @@
...
+ - The post menu is now only rendered on the root post when hovered upon.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:27]:
```suggestion
 - The post menu is now only rendered on the root post on hover over.
```

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -3,12 +3,58 @@
...
+#### Plugins
+ - Added an ``OnInstall()`` plugin hook.
+ - Added an ``OnSendDailyTelemetry()`` plugin hook.
+
+#### Bug Fixes
+ - Fixed an issue where the reply notification setting was still in effect even when Collapsed Reply Threads was enabled.
+ - Fixed an issue where running "mmctl config migrate" reset the configuration settings to defaults if the settings were already in the database.
+ - Fixed an issue where the custom status menu option was missing the "x" to clear status.
+ - Fixed an issue where the password reset link was valid for 1 hour instead of 24hours.
+ - Fixed an issue where the Mattermost import failed if an export contained a soft deleted team.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:47]:
```suggestion
 - Fixed an issue where the Mattermost import failed if an export contained a soft-deleted team.
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

File: source/install/cloud-changelog.md (hunk 2 above)
Comment (amyblais) [reply source/install/cloud-changelog.md]: Some screenshots here: https://github.com/mattermost/mattermost-webapp/pull/9683. When a new channel is created, there are some buttons and a channel intro text.

Comment (cwarnermm) [reply source/install/cloud-changelog.md]: Thank you! I wonder if we have an elegant way to refer to that part of the UI. @justinegeffen - do we have a standard way to refer to the UI that's visible when someone joins a brand new channel?

Comment (justinegeffen) [reply source/install/cloud-changelog.md]: > Thank you! I wonder if we have an elegant way to refer to that part of the UI. @justinegeffen - do we have a standard way to refer to the UI that's visible when someone joins a brand new channel?

Great question. I'd say "Channel landing page" except it's not really a page. Perhaps "Channel intro header" would work? Looking at the screenshot in the linked ticket I'd say that might work better?

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -3,12 +3,58 @@
...
+## Release 2022-02-16
+
+### Improvements
+
+#### User Interface (UI)
+ - Users are now able to use the channel switcher to search channels across teams if the users are part of more than one team.
+ - Updated the recommended minimum supported Firefox version to v91+.
+ - Clarified in-product error string "Oops!" as "Unable to continue" for both translators and target audiences in cases where a user doesn't have sufficient permissions to add users or guests.
+ - Removed incorrect in-product string text from the **Send full message contents** email notification option description displayed via **System Console > Site Configuration > Notifications**.
+ - Added the ability to send unsanitized user to the source user on ``user_updated`` event.
```
Comment (justinegeffen) [inline source/install/cloud-changelog.md:23]: Is there a word missing after "unsanitized user"?

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -3,12 +3,58 @@
...
+ - Added a **Create board** button to the Channel Intro header.
+ - The post menu is now only rendered on the root post on hover over.
+ - Updated a library used for storing drafts and other data in browser storage.
+ - Updated Playbooks version to 1.24.1.
+ - Update Boards version to 0.14.0.
+ - Enabled performance telemetry tracking for production deployments not running in developer mode. This telemetry tracking is disabled when telemetry is toggled off.
+
+#### Performance
+ - Reduced the number of menu components listening for keyboard and mouse events.
+ - Rerendering of ``CustomStatusEmoji`` component is now avoided on post hovering.
```
Comment (justinegeffen) [inline source/install/cloud-changelog.md:34]:
```suggestion
 - Re-rendering of ``CustomStatusEmoji`` component is now avoided on post hovering.
```

File: source/install/cloud-changelog.md [hunk 7]
```diff
@@ -3,12 +3,58 @@
...
 - [Release 2022-01-11](#release-2022-01-11)
-- [Release 2021-12-08](#release-2021-12-08)
+
+## Release 2022-02-16
+
+### Improvements
+
+#### User Interface (UI)
+ - Users are now able to use the channel switcher to search channels across teams if the users are part of more than one team.
+ - Updated the recommended minimum supported Firefox version to v91+.
```
Comment (esethna) [inline source/install/cloud-changelog.md:20]: Should this be a compatibility note rather than in the UI section

**Review by esethna — APPROVED** [review]

**Pull request merged**